app/models/manager_refresh/inventory_collection_default.rb

```diff
@@ -1,2 +1,98 @@
 class ManagerRefresh::InventoryCollectionDefault
+  class << self
+    def vms(extra_attributes = {})
+      attributes = {
+        :model_class            => ::Vm,
+        :association            => :vms,
+        :delete_method          => :disconnect_inv,
+        :attributes_blacklist   => [:genealogy_parent],
+        :use_ar_object          => true, # Because of raw_power_state setter and hooks are needed for settings user
+        # TODO(lsmola) can't do batch strategy for vms because of key_pairs relation
+        :saver_strategy         => :default, # Hooks are needed for setting user
+        :batch_extra_attributes => [:power_state, :state_changed_on, :previous_state],
+        :builder_params         => {
+          :ems_id   => ->(persister) { persister.manager.id },
+          :name     => "unknown",
+          :location => "unknown",
+        }
+      }
+
+      attributes.merge!(extra_attributes)
+    end
+
+    def miq_templates(extra_attributes = {})
+      attributes = {
+        :model_class            => ::MiqTemplate,
+        :association            => :miq_templates,
+        :delete_method          => :disconnect_inv,
+        :attributes_blacklist   => [:genealogy_parent],
+        :use_ar_object          => true, # Because of raw_power_state setter
+        :batch_extra_attributes => [:power_state, :state_changed_on, :previous_state],
```

Comment:
in my PR I added here
```:saver_strategy         => :default, # Hooks are needed for setting user```

we need to disable the batch saving now. Hooks are adding a user to each Template and some background jobs were failing without it. I need to look into what we are doing there, since the user setup is pretty cryptic (it's not really a user, since it runs as a background job, so it's pretty weird)